src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -2051,7 +2035,8 @@ private DynamoLineGeometryModel3D CreateLineGeometryModel3D(HelixRenderPackage r
         {
             var lineGeometry3D = new DynamoLineGeometryModel3D()
             {
-                Geometry = HelixRenderPackage.InitLineGeometry(),
+                //Do not set Geometry here
+                //Geometry = HelixRenderPackage.InitLineGeometry(),
```

Comment:
should we remove this?

Reply:
sure, I can remove these, and keep the comments only.